src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
I'd say you should be able to select anything in full view *only*, to get a 'preview' of sorts. In compact view, to me at least, you'd only be searching to add it to favourites.